### PR TITLE
Fix plots

### PR DESCRIPTION
**Pull request by felipeaoli**
Changes in layout and a massive number of changes in snr in order to work when one or more object type doens't produce snr output.

### CODE REVIEW

File: backend/framework/qlf/dashboard/bokeh/qaxwsigma/main.py [hunk 1]
```diff
@@ -41,13 +42,18 @@
 try:
     lm = get_scalar_metrics(selected_process_id, cam)
     metrics, tests  = lm['results']['metrics'], lm['results']['tests']
+    print(metrics.keys())
```
Comment (felipelm) [inline backend/framework/qlf/dashboard/bokeh/qaxwsigma/main.py]: Do you need to print this still?

File: backend/framework/qlf/dashboard/bokeh/qaxwsigma/main.py [hunk 2]
```diff
@@ -381,18 +388,64 @@ def histpar(yscale, hist):
...
 
 p_hist_w.quad(top= histval, bottom=bottomval, left='left', right='right',
        source=source_hist, 
         fill_color="dodgerblue", line_color="black", alpha=0.8,
        hover_fill_color='blue', hover_line_color='black', hover_alpha=0.8)
 
+#--------------------------------------------------------------
+# vlines ranges:
+bname = 'XWSIGMA'
+print(par[bname+'_NORMAL_RANGE'])
```
Comment (felipelm) [inline backend/framework/qlf/dashboard/bokeh/qaxwsigma/main.py]: Print really required?

File: backend/framework/qlf/dashboard/bokeh/qaxwsigma/main.py (hunk 1 above)
Comment (felipeaoli) [reply backend/framework/qlf/dashboard/bokeh/qaxwsigma/main.py]: Not at all.

File: backend/framework/qlf/dashboard/bokeh/qaxwsigma/main.py (hunk 2 above)
Comment (felipeaoli) [reply backend/framework/qlf/dashboard/bokeh/qaxwsigma/main.py]: Not at all. The print will be removed

**Pull request merged**